gpytorch/variational/orthogonal_decoupled_variational_strategy.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import torch
+
+from ..distributions import MultivariateNormal
+from ..utils.memoize import cached
+from ._variational_strategy import _VariationalStrategy
+
+
+class OrthogonalDecoupledVariationalStrategy(_VariationalStrategy):
```

Comment:
nit: if we've come so far as to spell our `Orthogonal`, maybe we should pay the extra two characters to add `ly`

Reply:
Done